migration/20170713-13-move-authentication-configuration-to-external-integrations.py

```diff
@@ -104,8 +104,8 @@ def convert_clever(_db, integration, provider):
         integration.setting(OAuthAuthenticationProvider.TOKEN_EXPIRATION_DAYS
         ).value = expiration_days
     
-configuration = Configuration.load()
-if not configuration:
+Configuration.load()
+if not Configuration.instance:
```

Comment:
This works fine, but on core at this point, `Configuration.load()` is supposed to be returning the configuration instance. Is that not happening?